decent-dao/fractal-interface
# Bugfix proposal state not updating consistently

## Description



## Notes



## Issue / Notion doc (if applicable)



## Testing
- Create/Use a Token DAO with 0 queue period
- Create a proposal and go through the motions
- After Queuing should see it update to execute button correctly


## Screenshots (if applicable)
